### PR TITLE
Added QuickSortDijsktra.java

### PR DESCRIPTION
**Pull request by the-cybersapien**
Added Java implementation for QuickSort using Dijsktra's 3-way partitioning.

### CODE REVIEW

File: Sorting Algorithms/Quick Sort Algorithm/Java/QuickSortDijsktra.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        }
+
+        System.out.println();
+    }
+
+    public static void quickSortDijsktra(int[] array, int low, int high){
+    	//Break condition for the Recursive Code
+    	if (low >= high) {
+    		
+    		return;
```
Comment (VamsiSangam) [inline Sorting Algorithms/Quick Sort Algorithm/Java/QuickSortDijsktra.java:47]: Requesting minor change. You don't need a new line if ```return``` statement is the only statement in the block. I'll add it to the conventions.

File: Sorting Algorithms/Quick Sort Algorithm/Java/QuickSortDijsktra.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    	}
+
+    	int lt = low;
+    	int ht = high;
+    	int pivot = array[low];
+
+    	for (int i = low + 1; i <= high; i++) {
```
Comment (VamsiSangam) [inline Sorting Algorithms/Quick Sort Algorithm/Java/QuickSortDijsktra.java:55]: Please add more explanatory comments inside this loop.

File: Sorting Algorithms/Quick Sort Algorithm/Java/QuickSortDijsktra.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+Code Contributor - Aditya Aggarwal (github.com/the-cybersapien, cybersapien.xyz)
+
+===== ===== ===== */
+
+import java.util.Scanner;
+
+public class QuickSortDijsktra{
+
+	public static void main(String[] args) {
+        //Initialize the Scanner Object to take input
```
Comment (VamsiSangam) [inline Sorting Algorithms/Quick Sort Algorithm/Java/QuickSortDijsktra.java]: Please add a space after ```//```, so your comment must go like ```// Initialize the Scanner Object to take input```. I'll update the conventions.

**Comment by VamsiSangam** [comment]
Sorry for coming up with new conventions on-the-fly, I thought it was okay not to list out all the conventions in CONVENTIONS.md. Will update it after your PR. And could you please update your other merge sort and quick sort Java codes to follow the new conventions?

**Comment by the-cybersapien** [comment]
I'll update the other two in a different PR. That'll help me keep track.
For now, please verify this Code.

**Comment by the-cybersapien** [comment]
Please update the CONVENTIONS.md file.
I'll update the other codes accordingly.

**Comment by VamsiSangam** [comment]
Your code is perfect! Will merge the PR now. You should be able to see your code live on the [quick sort](http://theoryofprogramming.com/2015/01/20/quick-sort-algorithm/) post soon. Please sync your fork once I add minor additions to the CONVENTIONS. Yes, a separate PR for updating conventions to your previous contributions would be perfect 😄

**Review by VamsiSangam — APPROVED** [review]

**Pull request merged**